packages/stardog-language-utils/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const ARBITRARILY_LARGE_NUMBER = 100000000000000;
```

Comment:
any reason not to use Number.MAX_SAFE_INTEGER?

Reply:
Not that I know of. I just moved this from where it previously existed, but changing it probably makes sense. I don't _think_ there is any _addition_ to the number (which would be problematic), but I'm not totally sure about that.

Reply:
Doesn't really matter. Was just curious.